segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/EpochInfo.java

```diff
@@ -62,10 +67,12 @@ protected void declareVersions() {
 
         private void write00(EpochInfo object, RevisionDataOutput output) throws IOException {
             output.writeCompactLong(object.epoch);
```

Comment:
As this is adding a new feild, it should be in write01 and read01, and the revision should be updated to 1.